metricbeat/docs/modules/http.asciidoc

```diff
@@ -37,6 +37,7 @@ metricbeat.modules:
   #method: "GET"
   #request.enabled: false
   #response.enabled: false
+  #response.is_array: false
```

Comment:
I was thinking again about the naming and I think we should go here with `json.is_array`. It's the json metricset so a prefix is needed, and as it's always about processing of the body / response anyways, we don't need this in.

Reply:
`json.is_array` makes sense to me. Will amend the change.